pkg/operation/botanist/component/kubeapiserver/kube_apiserver_test.go

```diff
@@ -1998,8 +2002,20 @@ rules:
 						Expect(kubeconfig.AuthInfos[0].AuthInfo.Token).NotTo(BeEmpty())
 					})
 
+					It("should not generate a kubeconfig secret for the user when EnableAdminKubeconfig is set to false", func() {
```

Comment:
Since EnableAdminKubeconfig is optional can we add one more test where if it not passed it should be defaulted to true?